src/main.c

```diff
@@ -66,6 +66,12 @@ USAGE \
 "                                <digest> is the X509 certificate's sha256 sum.\n" \
 "                                This option can be used multiple times to trust\n" \
 "                                several certificates.\n" \
+"  --insecure-ssl                Do not disable insecure SSL protocols/ciphers.\n" \
+"                                If your server requires a specific cipher, consider\n" \
+"                                using --cipher-list instead.\n" \
+"  --cipher-list=<list>          openssl ciphers to use. If default does not work\n" \
+"                                you can try openssl s_client to test alternative\n" \
+"                                ciphers.\n" \
```

Comment:
Tried to reword a bit, tell me if it sounds better (we're getting close to the 4k limit for a single string btw, whoever adds the next option will get extra fun splitting help into two strings!)

Also made --cipher-list work even if --insecure-ssl is used e.g. insecure-ssl only disables default cipher list, not explicit ones